dts/bindings/opamp/opamp-controller.yaml

```
@@ -0,0 +1,77 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Operational Amplifier (opamp)
+
+include:
+  - name: base.yaml
```

```suggestion
  - name: base.yaml
  - name: pinctrl-device.yaml
```

I think all internal opamp devices will have at least one pin routable to outside world, otherwise there is no meaning to have opamp integrated. Therefore adding pinctrl-device.yaml would make more sense here either in each SoC separately.